module-system/sov-modules-macros/src/module_info.rs

```diff
@@ -125,18 +130,19 @@ impl<'a> StructDef<'a> {
 
         Ok(quote::quote! {
             use sov_modules_api::AddressTrait;
```

Comment:
It wasn't touched in this PR, but still:

```suggestion
            use ::sov_modules_api::AddressTrait;
```